Sources/hostmgr/commands/vm/VMStart.swift

```diff
@@ -56,14 +57,17 @@ struct VMStartCommand: AsyncParsableCommand {
                 return
             }
 
+            let ipAddress: IPv4Address
             if waitForever {
                 let timeout = Duration.seconds(1_000_000_000) // Doesn't crash like .greatestFiniteMagnitude
-                try await vmManager.waitForVMStartup(for: configuration, timeout: timeout)
+                ipAddress = try await vmManager.waitForVMStartup(for: configuration, timeout: timeout)
```

Comment:
This is unlikely to work in a case where the machine needs to be set up (for instance, if we're making a new VM from scratch). I'd suggest double-checking that case, because `waitForVMStartup` relies on port 22 being open (which it wouldn't be on a brand-new VM)

Reply:
Ah, good point, will double-check that case.

Just to confirm, if that's the case, that is not new to my change though, right? Since with `--wait-forever` on a new VM from scratch, it's always been the case that we'd never return from `waitForVMStartup` anyway (even before my PR)… so I think that behavior is expected with `--wait-forever` anyway?

Reply:
I've [updated the instructions for when creating a new OS Template](https://github.com/Automattic/buildkite-ci/pull/380/commits/835b7d8807f90af8c8ab6a3454616e905fe1e70b) as part of [this PR](https://github.com/Automattic/buildkite-ci/pull/380), and confirmed by testing that, since the expected behavior is to call `vm create` first, then `vm start --persistent --wait-forever` in those cases, then it's expected that the `waitForVMStartup` will never return (and `vm start` will never finish and return prompt) in that particular case.

---

We _could_ potentially print the IP address _before_ waiting for SSH—by calling `try await ipAddress(forVmWithName: self.handle)` and printing the result _before_ the call to `waitForVMStartup`—so that the IP is shown even if we're configuring a VM from scratch that doesn't have SSH running yet. But:
 - That would have us call `ipAddress(forVmWithName:)` twice; or alternatively extract that call out of `waitForVMStartup`, but I don't like the semantics of separating those
 - The main goal of having the IP address printed is so we can SSH into the VM. While when we configure a brand new OS template from scratch, the only way to view the VM is via the "View" button of `hostmgr-helper` anyway. Printing the IP address in the console wouldn't help much in that particular case.

So instead I decided to update the wording of the log to `VM SSH IP Address: \(ipAddress)` to make it clear that this also implies that SSH is ready within the VM.